components/runtime/src/extensions/discord.rs

```diff
@@ -379,6 +380,29 @@ pub async fn op_discord_edit_message(
         .into())
 }
 
+#[op]
+pub async fn op_discord_crosspost_message(
+    state: Rc<RefCell<OpState>>,
+    channel_id: Id<ChannelMarker>,
+    message_id: Id<MessageMarker>,
+) -> Result<(), AnyError> {
+    let ctx = get_rt_ctx(&state);
+    match get_guild_channel(&state, &ctx, channel_id).await? {
+        Ok(_) => (),
+        Err(e) => return Err(e),
+    }
```

Comment:
preeeety sure this dosen't compile, the '?' operator does a early return on Err and unwraps the Ok value in case you didn't know

Reply:
Fixed, it didn't flag as an error for me, most likely an editor issue.